javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -69,7 +72,7 @@ import org.junit.Test
 @Suppress("EXPERIMENTAL_API_USAGE", "UNCHECKED_CAST")
 open class HandleManagerTestBase {
     @get:Rule
-    val log = LogRule()
+    val log = LogRule(Log.Level.Error)
```

Comment:
The full Arcs logging firehose is hard for me to digest, so I usually just scatter my own custom printlns around, that's what I've done here.